### PR TITLE
[13.0][FWD] #458

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Review by simahawk — APPROVED** [review]

File: shopinvader_search_engine/tests/test_action_server.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        variant_length = len(bindings.mapped("shopinvader_variant_ids"))
+        action = self.env.ref(
+            "shopinvader_search_engine.action_recompute_shopinvader_product"
+        )
+        action_context = action.with_context(
+            active_model="product.template",
+            active_ids=bindings.mapped("record_id").ids,
+        )
+        action_context.run()
+        self.assertEqual(job.count_created(), variant_length)
```
Comment (rousseldenis) [inline shopinvader_search_engine/tests/test_action_server.py:23]: Did a change here, if shopinvader.product contains more than than one variant (as jobs are created on them)

File: test-requirements.txt [hunk 2]
```diff
@@ -8,3 +8,4 @@ vcrpy-unittest
 openupgradelib @ git+https://github.com/OCA/openupgradelib.git
 unittest2 # For shopinvader test_controller, which inherits component
 odoo-test-helper
+odoo13-addon-queue-job @ git+https://github.com/OCA/queue.git@refs/pull/236/head#subdirectory=setup/queue_job
```
Comment (simahawk) [inline test-requirements.txt]: @rousseldenis can you drop this? Then we can merge!

Comment (rousseldenis) [reply test-requirements.txt]: @simahawk Done

**Pull request merged**